rusk-wallet/Makefile

```diff
@@ -10,7 +10,9 @@ install: build
 clippy: 
 	cargo clippy --all-features --release -- -D warnings
 
-test: build ## Run wallet tests
-	cargo test --release
+test: ## Run wallet tests
+	@rm -f /tmp/example.state
+	$(MAKE) -C .. prepare-dev
+	@../scripts/run-rusk-wallet-tests.sh
```

Comment:
Having this `end-to-end` test will cause a failure in case multiple PR are tested concurrently on our CI due to the same port used by the rusk binary